### DISCUSSION TITLE
v1.0.0-beta2

### DISCUSSION THREAD

**Discussion by Mrs4s** [Mrs4s/go-cqhttp#805] [Show and tell]
新增: 将配置文件类型从 `hjson` 切换为 `yml`, 可读性提高. by @wdvxdr1123 
新增: 支持使用手Q扫码处理滑条验证, 无需手动抓包 `ticket` 了.
新增: 在大部分情况下无需任何键盘输入即可登录了.
新增: 支持自定义 `[CQ:reply]` 的相关字段数据. by @sam01101 
新增: 支持通过 `name` 字段覆盖 `[CQ:at]` 发送时的名称. by @sam01101 
新增: 支持配置自动删除 `reply` 后紧跟的 `at` 消息了. by @sam01101 #764 
新增: 支持了更多临时会话来源消息的接收. #746 
新增: 支持回复来自 `QQ咨询` 的临时会话了.  
新增: 在会话过期时自动删除 `session.token`.
新增: `get_group_info` API支持 `no_cache` 参数了. by @sam01101 
新增: 为 `group_info` 增加了更多字段. by @sam01101 

修复: 在会话过期后, 登录会崩溃的问题. 
修复: 部分情况下 `ocr_image` API会出现错误的问题.
修复: `http api` 鉴权逻辑错误的问题. by @wdvxdr1123 #768 
修复: 登录完成后 `qrcode.png` 不会自动删除的问题.
修复: `get_group_forward_msg` API在获取不存在的消息ID时会报错的问题. by @sam01101 
修复: 等待密码加密密钥输入超时后不会还原控制台状态的问题. by @Ink-33 #794 
修复: `[CQ:image]` 的 `url` 字段会被错误的编码的问题. by @wdvxdr1123 
修复: 快捷回复时默认添加 `at_sender` 的问题. by @Yiwen-Chan  #774 

<hr /><em>This discussion was created from the release <a href='https://github.com/Mrs4s/go-cqhttp/releases/tag/v1.0.0-beta2'>v1.0.0-beta2</a>.</em>

**Comment by anye-star** [comment]
“将配置文件类型从 hjson 切换为 yml, 可读性提高”
能否提供一个兼容方案，使得原来的hjson文件得以保留且不需要重新编辑新的配置文件。

**Reply by Ink-33** [reply]
改成yml并不只是换了一个格式，这次没做转换器的一个原因是因为改动太大不太方便

**Comment by yestoday-tv** [comment]
改成yml后各种失败，我还是回去用1.0Alpha吧
